dubbo-common/src/main/java/org/apache/dubbo/common/utils/StringUtils.java

```diff
@@ -508,6 +508,14 @@ public static boolean isContains(String values, String value) {
         return isNotEmpty(values) && isContains(COMMA_SPLIT_PATTERN.split(values), value);
     }
 
+    public static boolean isContains(String str, char ch) {

```

Comment:
`contains` seems to be a better name to me

Reply:
@satansk I use `isContains` because of there has already two methods named `isContains` in `StringUtils` and `isXXX` is seems to be a better name for `XXXUtils`